rotkehlchen/db/upgrades/v36_v37.py

```diff
@@ -42,25 +42,46 @@ def _update_history_events_schema(write_cursor: 'DBCursor', conn: 'DBConnection'
 
         for entry in read_cursor:
             if entry[11] is None:
-                new_entries.append(entry[:11] + ('none',) + entry[12:])
+                new_entries.append(entry[:11] + ('none',))
```

Comment:
Could you write in a comment what this is doing? I came here and I have 0 context about what was happening here and I still don't know 100%

Reply:
There is a docstring, but sure, I'll add a few comments